### PR TITLE
Add MacOS (homebrew) build instructions

### PR DESCRIPTION
**Pull request by tmc**

### CODE REVIEW

**Comment by niemeyer** [comment]
Nice, so it does work fine there?

**Comment by tmc** [comment]
Yes

File: README.md [hunk 1]
```diff
@@ -35,15 +35,27 @@ Requirements
...
+
+Ubuntu:
 
     sudo add-apt-repository ppa:ubuntu-sdk-team/ppa
     sudo apt-get update
     sudo apt-get install ubuntu-sdk qtbase5-private-dev qtdeclarative5-private-dev
 
+MacOS:
+
+    brew install gcc48 qt5
```
Comment (aybabtme) [inline README.md:58]: `gcc48` won't be there by default, you need to `brew tap homebrew/versions` first, then you can `brew install gcc48`.

Comment (tmc) [inline README.md:58]: Thanks

**Comment by kisielk** [comment]
I can confirm that I was able to build qml and run the particles example on OS X 10.8 with these instructions.

**Comment by aybabtme** [comment]
Same here.

Cheers,

## 

Antoine Grondin
*
*

_Live long and prosper_

**Comment by marconi** [comment]
Works here too. OSX 10.8.5

**Comment by scottferg** [comment]
Working here

**Comment by niemeyer** [comment]
Thanks all. I'll integrate these instructions today.

**Pull request merged**